types/lib/model.d.ts

```diff
@@ -1975,7 +1980,7 @@ export abstract class Model<TModelAttributes extends {} = any, TCreationAttribut
   public static create<M extends Model>(
     this: ModelStatic<M>,
     values: M['_creationAttributes'],
-    options: CreateOptions<M['_attributes']> & { returning: false }
+    options: Omit<CreateOptions<M['_attributes']>, 'returning'> & { returning: false }
```

Comment:
Should we give `ignoreDuplicates` similar treatment?

A conflict with "do nothing" could potentially end up in the instance not being saved to the database. Might be better to invalidate the response (`void`) and force a reload of the instance if necessary

Reply:
Actually I like that, let's do it. Better to be correct on the safe side.